modules/http4s/src/main/scala/ciris/http4s/http4s.scala

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright 2017-2022 Viktor Lövgren
+ *
+ * SPDX-License-Identifier: MIT
+ */
+
+package ciris
+
+import org.http4s.Uri
+import org.tpolecat.typename.typeName
+
+package object http4s {
+  implicit final val uriConfigDecoder: ConfigDecoder[String, Uri] =
+    ConfigDecoder[String].mapOption(typeName[Uri])(Uri.fromString(_).toOption)
```

Comment:
Ignoring the real parsing error, as it is seems totally enough and more appropriate (to me) to provide `ConfigError.decode` in case of failure.
But I am not against using/including original error info.

Reply:
Sounds good to leave it as is for now. We can always change it later if we want.